openerp/addons/base/ir/ir_qweb.py

```diff
@@ -1277,38 +1288,11 @@ def css_page(self, page_number):
                     pages.append([rule])
                     page = pages[-1]
                     page_selectors = selectors
-            if len(pages) == 1:
-                pages = []
             for idx, page in enumerate(pages):
-                self.set_cache("css.%d" % (idx+1), ''.join(page))
-            content = '\n'.join(css_url % i for i in range(1,len(pages)+1))
-            self.set_cache("css.0", content)
```

Comment:
Because you loop on each existing attachemnts: https://github.com/odoo/odoo/pull/8315/files#diff-3ad2ad1e9b88257a01a42d04ca1c0605R1189